web/src/components/task/task-sidebar-data/task-sidebar-data.tsx

```diff
@@ -45,6 +45,9 @@ export const TaskSidebarData: FC<TaskSidebarDataProps> = ({
 
     return (
         <div className={styles['task-sidebar-data']}>
+            {isCategoryDataEmpty && (
+                <Text cx={styles.text}>{`The selected category doesn't have data attributes`}</Text>
```

Comment:
Why are you using a template literal here?

Reply:
Done